docs/development.md

```diff
@@ -71,28 +69,35 @@ Everytime you want to run bors:
 - Try `@bors ping` on some PR on the test repository :)
 
 ## Database
-You must have `sea-orm-cli` installed for the following commands to work.
+You must have `sqlx-cli` installed for the following commands to work.
 ```console
-$ cargo install sea-orm-cli
+$ cargo install sqlx-cli --no-default-features --features native-tls,postgres
 ```
 
-You must also set up a `DATABASE_URL` environment variable. **You can use SQLite for local testing,
-but when entities are regenerated, it should be done against a Postgre database!**
+The database can be set up with the docker-compose file in the root of the repository:
+
+```console
+$ docker-compose up -d
+```
+
+Then, set the `DATABASE_URL` environment variable to the connection string of the database. The content of the variable is can be found in the `.env.example` file.
```

Comment:
Let's also tell the user that if `.env` is present, it will  be automatically picked up by `sqlx`.